adodb.inc.php

```diff
@@ -658,6 +659,29 @@ function Time() {
 		return false;
 	}
 
+	/**
+	 * Parses the hostname to extract the port.
+	 * Overwrites $this->host and $this->port, only if a port is specified.
+	 * The Hostname can be fully or partially qualified,
+	 * ie: "db.mydomain.com:5432" or "ldaps://ldap.mydomain.com:636"
+	 * Any specified scheme such as ldap:// or ldaps:// is maintained.
+	 *
+	 * @return true
+	 */
+	function parseHostNameAndPort() {
```

Comment:
This is really an internal function, I don't see any benefit of a public method. Suggest to make it _protected_.

Reply:
I'm fine with that too.